lib/elixir_analyzer/exercise_test.ex

```diff
@@ -35,7 +35,7 @@ defmodule ElixirAnalyzer.ExerciseTest do
     feature_test_data = Module.get_attribute(env.module, :feature_tests)
     assert_call_data = Module.get_attribute(env.module, :assert_call_tests)
     check_source_data = Module.get_attribute(env.module, :check_source_tests)
-    suppress_tests = Module.get_attribute(env.module, :suppress_tests, [])
+    suppress_tests = Module.get_attribute(env.module, :suppress_tests, []) || []
```

Comment:
I'm not sure if this is the best solution, but filter_suppressed_results/1 in lib/elixir_analyzer/exercise_test.ex expects it to be a list.

Reply:
Ah. In that case the old default value is useless I think:
```suggestion
    suppress_tests = Module.get_attribute(env.module, :suppress_tests) || []
```